drivers/led_strip/Kconfig.ws2812

```diff
@@ -7,27 +7,16 @@
 #
 # https://wp.josh.com/2014/05/13/ws2812-neopixels-are-not-so-finicky-once-you-get-to-know-them/
 
-menuconfig WS2812_STRIP
-	bool "WS2812 (and compatible) LED strip driver"
-	help
-	  Enable LED strip driver for daisy chains of WS2812-ish (or WS2812B,
-	  WS2813, SK6812, Everlight B1414, or compatible) devices.
-
-choice WS2812_STRIP_DRIVER
-	prompt "Driver backend"
-	default WS2812_STRIP_SPI
-	depends on WS2812_STRIP
-
 config WS2812_STRIP_SPI
 	bool "SPI driver"
-	depends on SPI
+	select SPI
 	help
 	  The SPI driver is portable, but requires significantly more
 	  memory (1 byte of overhead per bit of pixel data).
 
 config WS2812_STRIP_I2S
 	bool "I2S driver"
-	depends on I2S
+	select I2S
 	help
 	  Uses the I2S peripheral, memory usage is 4 bytes per color,
 	  times the number of pixels. A few more for the start and end
```

Comment:
These could be removed as well if you use following example:
```
	select I2C if $(dt_compat_on_bus,$(DT_COMPAT_BOSCH_BME680),i2c)
	select SPI if $(dt_compat_on_bus,$(DT_COMPAT_BOSCH_BME680),spi)
```

See also use of `DT_ANY_INST_ON_BUS_STATUS_OKAY` in drivers/sensors

Reply:
Thank you for inform. I fixed it.